extendr-api/tests/extendr_module_tests.rs

```diff
@@ -0,0 +1,28 @@
+//! Tests for [`extendr_module`] procedural macro.
+//!
+mod root {
```

Comment:
I'm absolutely fucking stupid.
So I need the fix for #469, otherwise extendr is too annoying to work with.
Once that one is merged, I'll rebase this, and then these commits will disappear.
Does that work for you?